practical-penguins/trivia_tavern/trivia_builder/models.py

```diff
@@ -0,0 +1,24 @@
+from django.db import models
+from django.contrib.auth.models import User
+from django.urls import reverse
+
+
+class TriviaQuiz(models.Model):
+
+    name = models.CharField(max_length=30, blank=False,)
+    author = models.ForeignKey(User, on_delete=models.CASCADE)
+    description = models.CharField(max_length=500,)
+    date_posted = models.DateTimeField(auto_now_add=True)
+
+    def __str__(self):
+        return self.name
```

Comment:
Decent usage of `__str__`, although it wouldn't hurt to use something that more directly identifies it as a quiz, e.g.

```py
def __str__(self):
     return "TriviaQuiz: "+self.name"
     # alt: f"TriviaQuiz<{self.name}>"
```

To further help uniquely identify it, you could also do something like author + name, although that's probably not needed in your case (it would be well suited for a `__repr__`).